.github/workflows/linux-check.yaml

```diff
@@ -95,11 +95,11 @@ jobs:
 
   linux-matrix:
     name: Linux builds and tests
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     strategy:
       fail-fast: false
       matrix:
-        compiler: [{ CXX: g++-12, CC: gcc-12 }, { CXX: clang++-14, CC: clang-14 }]
+        compiler: [{ CXX: g++-12, CC: gcc-12 }, { CXX: clang++-16, CC: clang-16 }]
```

Comment:
```suggestion
        compiler: [{ CXX: g++-14, CC: gcc-14 }, { CXX: clang++-16, CC: clang-16 }]
```

Reply:
Then I have also increased clang to the highest version.